sdk/lib/rtl/runonce.c

```diff
@@ -0,0 +1,130 @@
+
+/* PSDK/NDK Headers */
```

Comment:
rtl uses `rtl.h` as a PCH, so you should include that here to avoid differences between PCH and non-PCH builds.

Reply:
Fixed.